DESCRIPTION

```diff
@@ -25,6 +25,7 @@ Depends:
     graphics,
     R (>= 3.0.0)
 Imports:
+    lifecycle,
```

Comment:
I think because of this we also need to import a single function from this package. please add a corresponding roxygen line to https://github.com/Roche/crmPack/blob/main/R/crmPack-package.R
(otherwise R CMD check will complain)

Reply:
see e.g. https://github.com/insightsengineering/hermes/blob/5ad69c62b0b7dec08e23829dffbcb4d8be1d5ca6/R/package.R#L17 for comparison

Reply:
Sure, thank you for spotting this!